startup.krun

```diff
@@ -42,6 +42,7 @@ def patch_runner(vm_def):
         assert "iterations_runner." in vm_def.iterations_runner
         _, suffix = vm_def.iterations_runner.rsplit(".", 1)
         vm_def.iterations_runner = os.path.join(STARTUP_RUNNER_DIR, "startup_runner." + suffix)
+    vm_def.instrument = False
```

Comment:
We can consider startup and instrumentation modes to be mutually exclusive. Maybe this should be an `assert` or something? [I'm not saying I have the answer here... just trying to work out what the right thing is!]